Author: Reiner Herrmann <reiner@reiner-h.de>
Description: Use system aapt instead of trying to use included one

--- a/brut.apktool/apktool-lib/src/main/java/brut/androlib/ApkOptions.java
+++ b/brut.apktool/apktool-lib/src/main/java/brut/androlib/ApkOptions.java
@@ -29,5 +29,5 @@
 
     public String frameworkFolderLocation = null;
     public String frameworkTag = null;
-    public String aaptPath = "";
+    public String aaptPath = "/usr/bin/aapt";
 }
